GANDLF/cli/deploy.py

```diff
@@ -86,23 +86,26 @@ def run_deployment(
 
 
 def deploy_docker_mlcube(
-    mlcubedir,
-    outputdir,
-    entrypoint_script=None,
-    config=None,
-    modeldir=None,
-    requires_gpu=None,
-):
+    mlcubedir: str,
+    outputdir: str,
+    entrypoint_script: str = None,
```

Comment:
In the cases when the param is typed to string and default set to None, as in the previous cases it should be Union[str,None]

Reply:
Fixed with `Optional`.